docs/credential-plugins.rst

```diff
@@ -379,3 +380,66 @@ Below shows an example of a configured Thycotic Secret Server credential.
 
 .. image:: _static/images/credentials-create-thycotic-server-credential.png
    :alt: Example new Thycotic Secret Server credential lookup dialog
+
+
+.. _ug_credentials_github_app_lookup:
+
+GitHub App Token Lookup
+~~~~~~~~~~~~~~~~~~~~~~~
+
+This plugin allows a GitHub app token to be used as a credential input source to pull secrets from GitHub App. AWX uses existing GitHub auth from organizations' GitHub repos. Refer to `Generating an installation access token for a GitHub App <https://docs.github.com/en/apps/creating-github-apps/authenticating-with-a-github-app/generating-an-installation-access-token-for-a-github-app>`_ for more detail.
+
+1. Create a lookup credential that stores your secrets. See TBD for detail.
+..(link available in separate PR that has this information).
+
+2. When **GitHub App Installation Access Token lookup** is selected for **Credential Type**, provide the following attributes to properly configure your lookup:
+
+- **GitHub App ID** (required): provide the app ID used for communicating with your GitHub App
+- **GitHub App Installation ID** (required): ID of the installation that you want to authenticate as
+- **RSA Private Key** (required): provide the generated private key obtained by the GitHub organization which your repo resides. See `Managing private keys for GitHub Apps <https://docs.github.com/en/apps/creating-github-apps/authenticating-with-a-github-app/managing-private-keys-for-github-apps>`_
```

Comment:
```suggestion
- **RSA Private Key** (required): provide the private key that your GitHub instance generated. It can be obtained from the GitHub App maintainer within GitHub. See `Managing private keys for GitHub Apps <https://docs.github.com/en/apps/creating-github-apps/authenticating-with-a-github-app/managing-private-keys-for-github-apps>`_
```